CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 ### Changed
+- Updated civis.utils.run_template to accept kwargs. (#526)
```

Comment:
In the `### Deprecated` section, please add a note about the `JSONValue` kwarg of `civis.utils.run_template` being deprecated.